velox/exec/Task.cpp

```diff
@@ -78,6 +78,17 @@ std::string errorMessageImpl(const std::exception_ptr& exception) {
   return message;
 }
 
+// Add 'running time' metrics from CpuWallTiming structures to have them
+// available aggregated per thread.
+void addRunningTimeOperatorMetrics(exec::OperatorStats& op) {
+  op.runtimeStats["runningAddInputWallNanos"] =
```

Comment:
Hi Sergey, I figured they're scheduled time of an operator per driver. Do you think it makes sense to call them scheduledAddInputTimeInNanos? Otherwise, I'm okay with the change. Thanks for iterating on it!

Reply:
Since the runtime stats is ended with WallNanos, can we just name it  getOutputWallNanos addInputWallNanos? @mbasmanova what's the naming convention for metrics we use in Velox? WDYT?

Reply:
@xiaoxmeng 
We've got "blocked*" and "queued*" metrics. To see the full picture we are missing "running*" metrics.
Since we already have them split in three groups in static members of the operator stats (which are aggregated and thus fairly useless), I merely mimic them in metrics.
Adding "running" prefix makes it easier to locate among the other metrics and we have quite a few already.